### ISSUE TITLE
Bug play video

### ISSUE THREAD

**Issue by NicotherZ** [YePpHa/YouTubeCenter#1880]
Hi,
When I want to watch a video, it takes 20 or 30 seconds for the video to be loaded. (url : *.googlevideo.com long to be loaded).
Or i have to refresh the page, and it's done, when i desactivate youtubecenter, it works without waiting.

What's wrong?

Please help.

(sorry for my bad english :/)

**Comment by maiofalmo** [comment]
The whole thing is a disaster at the moment.

**Comment by LiamBM** [comment]
Yeah, not a lot working right now. Everything seems pretty buggy, and it's been forever since an update.

**Comment by killerz298** [comment]
I am using the latest version of Firefox with the latest developer version through greasemonkey. I have been experiencing the same issue for awhile now through many versions of the script and many firefox updates. As soon as a video starts to load things hang for about 30 seconds and it says waiting for *googlevideo.com etc.... Recently what happens is the html5 player loads (but I can't play anything) and then after the hang it switches to the flash player and plays normally. If I reload the page the flash player loads and plays immediately. I have tried force flash and aggressive flash in options with the same result. The issue is directly related to this option. When I set the option to default, there is no hang and I can immediately start playing with the loaded html5 player.
